pages/regions.js

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, Icon, P } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const redirect = router.query.redirect;
+
+  return (
+    <>
+      {data.regions.map((region) => {
+        const href = (redirect === 'login') ? region.url : region.altUrl;
+
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={href}>
+                <a>
+                  <Icon name={region.icon} subStyle={region.color} mainStyle="fill" />
+                </a>
+              </Link>
+            </div>
+            <P className='h6'>
+              <span style={`color:${region.color}`}>{region.title}</span>
```

Comment:
We should not be doing inline styles like this. There are some classNames for this in the typography SASS file:

https://github.com/jambonz/next-static-site/blob/main/src/styles/_typography.scss#L110-L120

So here you would instead do it like:

```js
<span className={`color-${region.color}`}>{region.title}</span>
```

The way you have it here the colors `blue` and `teal` aren't even the same color as the icons which use variables from the SASS.